packages/fxa-auth-server/test/client/api.js

```diff
@@ -650,6 +650,20 @@ module.exports = config => {
     );
   };
 
+  ClientApi.prototype.attachedClientDestroy = function (sessionTokenHex, clientData) {
+    return tokens.SessionToken.fromHex(sessionTokenHex)
+      .then(
+        (token) => {
+          return this.doRequest(
+            'POST',
+            `${this.baseURL  }/account/attached_client/destroy`,
```

Comment:
`${this.baseURL  }` what happened here?

Reply:
Huh, `api.js` seems to be full of these, for reasons I don't understand. I assume I copy-pasted it from elsewhere.  Will fix mine and then address the others in a followup. Nice catch!